### DISCUSSION TITLE
Exploring Collaboration with Akash Network for SparkTheSummer Hackathon

### DISCUSSION THREAD

**Discussion by fenilmodi00** [akash-network/community#571] [Governance Proposals]
Hello Akash Team,

I'm reaching out to propose a partnership between Akash Network and SparkTheSummer, a 36-hour hackathon scheduled for July 26-27 in Gujarat, India. Our event is designed to bring together over 400 developers, tech enthusiasts, and professionals to work on innovative projects and tackle real-world challenges within a short timeframe.

Given Akash Network's leading position in decentralized cloud computing, we believe a partnership could offer significant benefits to both parties. SparkTheSummer aims to foster creativity and collaboration, and we see Akash's unique approach to peer-to-peer computing resources as a perfect match for our hackathon. 

Here's how we envision Akash Network's involvement in SparkTheSummer:

- **Sponsorship**: Akash can sponsor our hackathon, providing resources and financial support to help us deliver a successful event.
- **Workshops and Mentoring**: We invite Akash experts to conduct workshops on decentralized cloud computing and to mentor participants during the hackathon.
- **Project Evaluation**: Akash representatives can join our judging panel, providing expert feedback to participants and evaluating their projects.

These elements align with Akash's commitment to innovation and community building. By engaging with our hackathon, Akash can connect with a vibrant group of developers and tech enthusiasts, and further promote its decentralized cloud platform.

We'd be delighted to discuss the details of this potential partnership and explore how Akash Network can contribute to SparkTheSummer's success. Please let me know if we can schedule a call to discuss further.

Thank you for your consideration, and I look forward to hearing from you soon.

Best regards,
Fenil Modi
Lead Organizer
SparkTheSummer Hackathon

**Comment by ghost** [comment]
Hey there! Thanks for sharing! 

Would be great if you add links to your social networks and profiles 

Have you done hackathons in the past? what can you say about that?

How much will this cost to Akash Network?

**Reply by fenilmodi00** [reply]
# Partnership Proposal with Akash Network

Hello Robert, 

Thanks for getting back to me!

You can find more information about SparkTheSummer on our website: [sparkthesummer.tech](http://sparkthesummer.tech/). We've also compiled all our social media profiles here: [linksta.cc/@STS](https://linksta.cc/@STS). This should give you a good sense of our community and previous events.

Regarding your other questions, I'm the community lead at GDSC UVPCE, a community with over 600 members. I've organized many events and local hackathon, and I was part of HackThisFall 2024, one of India's biggest hackathons. Now, we're taking our experience and passion to create SparkTheSummer as a national-level event.

The sponsorship cost for Akash Network would be $4,500. This investment will help us deliver a high-quality hackathon, and in return, Akash Network will gain visibility and the opportunity to engage with a large and enthusiastic developer community and many more things.

Best regards,
Fenil Modi  
Lead Organizer

**Comment by rodri-r** [comment]
This sounds interesting and it looks like it might draw some visibilty to Akash Network. 

What tracks would you expect devs to build that would help Akash Network?

Would those $4,000 be rewarded to winners of the Spark The Summer hackathon? or at least part of it?

How many participants would you expect this hackathon to have?

**Reply by fenilmodi00** [reply]
We're excited that you're interested in SparkTheSummer.

Regarding the tracks, developers will build projects in areas relevant to Akash Network, including decentralized cloud computing, Web3/blockchain, AI, and Med-Tech. These domains provide ample opportunity for innovation and align well with Akash's focus.

Concerning the $4,500 sponsorship, part of it will be awarded to the hackathon winners, with the remainder covering costs to organize the in-person event, like venue, logistics, and supplies. Akash's support will help us create a memorable experience for everyone involved.

We expect over 400 participants, with teams of 3-4 members each. Your contribution will help us engage a large and enthusiastic group of developers and tech enthusiasts.

**Comment by fenilmodi00** [comment]
### How We're Going to Market Our Hackathon

We will market our hackathon through the following channels:

- **Google's Community Sites**: Our event will be featured on various Google Developer Student Clubs (GDSC) community sites, including the GDSC chapters : [GDSC U. V. Patel College of Engineering](https://gdsc.community.dev/u-v-patel-college-of-engineering-mehsana/) and many more
- **Community Partners**: We have over 7 community partners who will assist in promoting the hackathon through their networks.
- **IEEE India**: IEEE India is one of our key community partners and will help us reach a wider audience through their extensive network.


### Benefits of In-Person Hackathons Over Online Hackathons

- **Enhanced Collaboration**: In-person hackathons allow for face-to-face interactions, fostering stronger teamwork and collaboration. Participants can brainstorm and work on projects together in real time.
- **Networking Opportunities**: These events provide unique opportunities to network with peers, mentors, and industry experts, which can lead to future collaborations and professional growth.
- **Immediate Feedback and Support**: On-the-spot mentoring and support from experts help participants overcome challenges more efficiently and enhance the learning experience.
- **Engaging Environment**: The energy and excitement of an in-person hackathon create a motivating and immersive environment, boosting creativity and drive.
- **Hands-On Experience**: Participants can engage in hands-on activities, such as hardware integration or product demos, which are more challenging to facilitate in an online setting.

**Reply by brewsterdrinkwater** [reply]
@fenilmodi00 

Thank you for continuing to add detail to this discussion. 

Couple of more questions: 

- Will there be a list of developers' emails that we can access as a sponsor?
- What kind of metrics are yall tracking for success?

**Reply by fenilmodi00** [reply]
## Discussion Update

### Developer Contact Information
Yes, there will be a list of developers' details. Recently, we had a call with the team at DoraHacks, and we are planning to host our IRL hackathon on their platform. This will help us in collecting registrations and facilitating evaluation and judging processes.

### Metrics for Success
Our metrics for success include:
- The number of hackers who participated in the IRL event
- The new projects they built that align with industry needs
- The skills and knowledge they gained by participating

We are also open to tracking other metrics that can help us improve. We would love to hear suggestions from the community on this.
@brewsterdrinkwater

**Comment by brewsterdrinkwater** [comment]
@fenilmodi00 Please let us know next steps, if the plan was to move forward.

**Reply by fenilmodi00** [reply]
@brewsterdrinkwater  Thank you for considering our involvement!

To move forward, we request the transfer of funds befor 2nd July and some marketing support from the Akash community. We currently have 179 in-person registrations and more in coming. We're adding an Akash track to our hackathon and will conduct a workshop on it.

**Reply by ghost** [reply]
Hey Fenil! 

What kind of marketing support are you referring to?
About the Akash track, what you guys have on mind? 
Is the Dorahacks site live for this hackathon?

**Reply by brewsterdrinkwater** [reply]
Thank you @fenilmodi00 

This was discussed at sig-community, and the goal of the hackathon aligns with the community. OCL will support sending an Akash community member, as well, to help answer questions on site, and help with the workshop. 

Funds have been sent: https://www.mintscan.io/akash/tx/84845AB0C3B0D8E8333F6B6ED2EFDA8655A3E6A34D05EE0370710A81B2A3F503?height=16983705

**Comment by fenilmodi00** [comment]
Hello Akash Community,

Thank you for supporting SparkTheSummer! Your partnership was crucial to the success of our hackathon. I am incredibly passionate about organizing in-person events, and I hope that the sleepless days of preparation were worthwhile. The event was a tremendous success, and we had an amazing time.

Here’s a snapshot of our event:

- **Total Applications**: 1,093
- **Submitted for Review**: 597
  - **Teams**: 48
  - **Individuals**: 321

On the event day, over 300 hackers gathered under one roof, creating an energetic and collaborative environment.

In the past, I organized an Akash watch party, so my local community members are familiar with Akash. However, by hosting such a significant hackathon, we have introduced Akash to a broader audience across India.

The surprise special prize pool for the Akash track on the event day was amazing! It added an exciting element to the hackathon and further motivated our participants.

A special thanks to Panther and his team for attending our hackathon. Their support as mentors and judges was invaluable, providing expert guidance and feedback to the participants.

For more details, you can explore our event pages:
- [Devfolio](https://sparkthesummer.devfolio.co/)
- [DoraHacks](https://dorahacks.io/hackathon/sparkthesummer)

You can also view the list of winners [here](https://docs.google.com/spreadsheets/d/1ILlI7AJekzUNrD220JPd877Hv_Gau4z-5UShPX5TzQk/edit?gid=0#gid=0).

Here are some glimpses of the hackathon; we’ll be adding more soon: [Event Photos](https://drive.google.com/drive/folders/14qYSMWTpdDUWl2mjtCjyWo-Opf7-uPjG).

Looking forward to hearing more from @HoomanDigital **Bold** | Panther and his team.

Best regards,  
Fenil Modi  
Lead Organizer  
SparkTheSummer Hackathon

**Reply by ghost** [reply]
Hey Fenil! Outstanding job!! feel free to share some of the event photos in the insiders channel and social media
